recap/converters/bigquery.py

```diff
@@ -69,7 +70,7 @@ def to_recap(self, fields: list[bigquery.SchemaField]) -> StructType:
                         scale=field.scale or 0,
                     )
                 case _:
-                    raise ValueError(f"Unrecognized field type: {field.field_type}")
+                    field_type = UnknownType()
```

Comment:
Maybe `UnknownType(field_type=field_type)`? I'm thinking we should try to give the dev as much info as possible about the thing we can't convert.